Fix Kap preventing sleep

karaggeorge opened the pull request:
When an editor was open, Kap would prevent the computer from going sleep. This was caused by chrome's functionality of preventing sleep when there's media playing on the page. Until a better solution is found, we decided to at least pause the editor window when it is blurred, and resume when focused.

This indeed removes Kap from the list of Apps that prevent sleep a couple of seconds after blurring. The issue, of course, will remain if you keep the editor focused.

Questions:
- While doing this, I was also wondering if it would be worth utilizing that functionality and preventing the computer from sleeping while aperture is recording. I think I've heard of a couple of issues of putting the computer to sleep while recording, but I'm not 100% sure.

sindresorhus commented:
If I pause the video, blur the window, and then focus the window again it starts playing, but it should be paused like it was before being blurred.

karaggeorge commented:
@sindresorhus 

Good catch, updated. Also, fixed the preferences categories color issue. It will need some more touching up (the gray background focus state) but I'm working on a follow up PR to add tab accessibility to the preferences window, so I'll address further there.

sindresorhus commented:
> * While doing this, I was also wondering if it would be worth utilizing that functionality and preventing the computer from sleeping while aperture is recording. I think I've heard of a couple of issues of putting the computer to sleep while recording, but I'm not 100% sure.

Better to do that at the source: https://github.com/wulkano/Aperture/pull/64
